controller.js

```diff
@@ -13,7 +13,34 @@ function getTopics(req, res, next){
     .catch(next)
     }
 
-function getArticle(req, res, next){
+    function getArticleComments(req, res,next){
+    const articleId = req.params.article_id
+    console.log(articleId)
+    if(isNaN(articleId)){
```

Comment:
this may not be working as you expect as articleId will come through on params as a string